templates/zerver/help/streams-and-private-messages.md

```diff
@@ -0,0 +1,96 @@
+# Streams and private messages
+![Streams and private messages overview](/static/images/help/streams-private-overview.png)
+
+On Zulip, you can easily communicate with other users using the **streams and private messaging** feature located at the bottom of your screen.
+
+Zulip supports both **private messaging** and group chats via conversation **streams**.
+
+**Streams** are either:
+* **Public** - Public streams are for open discussions. All users can subscribe to public streams and discuss there. Any Zulip user can join any public stream in the realm, and can view the complete message history of any public stream without joining the stream.
+* **Private** - Private streams are for confidential discussions and are only visible to users who've been invited to subscribe to them. Users who are not members of a private stream cannot subscribe to the stream or read or send messages to the stream.
+
+**Private messaging** can either be **one-on-one** or **group** conversations. They can only be seen by users participating in the message.
+
+## Creating a new stream message
+![New stream message](/static/images/help/new-stream-message.png)
+
+To submit a new stream message, first click the **New stream message** button located at the bottom of your screen.
+
+![New stream message](/static/images/help/new-stream.png)
+
+A box similar to the one shown in the previous image will appear.
+
+In the **Stream** field, input the stream name of the message you want your message to be located in. When you begin typing a stream name, autocomplete suggestions will appear.
+
+![Stream does not exist](/static/images/help/stream-none.png)
+
+If you attempt to input a nonexistent stream name, an error message will appear.
+
+Once you have entered your stream name, enter the topic for your message in the **Topic** field. Your topic name can be no longer than 52 characters.
+
+Finally, enter your message in the box labeled **Compose your message here...** or messaging box. You can now send your message to the specified stream under the specified topic.
+
+You can always delete your message by clicking the **x ![x](/static/images/help/x.png)** icon located at the top-right corner of your messaging box.
```

Comment:
Small change: `the **x** (![x](/static/images/help/x.png)) icon`, just for consistency with other documentation